### ISSUE TITLE
Git in Docker is not compatible with `git branch --show-current`

### ISSUE THREAD

**Issue by tsawada** [google/copybara#176]
git command included in the Docker image does not support `git branch --show-current` option (copybara started to use in 284ff952c).

The option requires git >= [2.22.0](https://github.com/git/git/blob/master/Documentation/RelNotes/2.22.0.txt#L32), while the docker image (built from the Dockerfile in this repository) has git 2.20.1.
```
$ docker build -t copybara . && docker run --rm copybara git --version
git version 2.20.1
```

Migration in between git.origin and git.destination fails with:
```
ERROR: Error executing 'git': Process terminated by signal 
```

Solution 1: Somehow install newer git package.
Solution 2: Use `git rev-parse --abbrev-ref HEAD`

**Comment by hsudhof** [comment]
Thank you for the contribution, let me take a look!

**Comment by hsudhof** [comment]
It looks like this breaks //copybara/integration:tool_test - test_git_iterative_same_repo

**Comment by tsawada** [comment]
The fix has been merged as 65e6de2f84baa9d9ffab5297ea15725c6ae34d62.
Thanks!
